ambry-replication/src/test/java/com/github/ambry/replication/ReplicationTest.java

```diff
@@ -1417,6 +1418,9 @@ public void blockDeprecatedContainerReplicationTest() throws Exception {
       }
       Set<MessageInfo> remoteMissingStoreKeys = replicaThread.getMissingStoreMessages(replicaMetadataResponseInfo, remoteNode, remoteReplicaInfo);
       assertEquals("All DELETE_IN_PROGRESS blobs qualified with retention time should be skipped during replication", 0, remoteMissingStoreKeys.size());
+      Map<StoreKey, StoreKey> remoteKeyToLocalKeyMap = replicaThread.batchConvertReplicaMetadataResponseKeys(response);
+      replicaThread.processReplicaMetadataResponse(remoteMissingStoreKeys, replicaMetadataResponseInfo, remoteReplicaInfo,
+          remoteNode, remoteKeyToLocalKeyMap);
```

Comment:
minor:  `checkSet` is never used, consider removing it?  (Also, format this file please)